inc/views/page_header.php

```diff
@@ -57,7 +57,7 @@ public function render_page_header( $context ) {
 			return;
 		}
 		$header_layout = get_theme_mod( 'neve_page_header_layout', 'normal' );
-		if ( $header_layout !== 'normal' && $context === 'single-page' ) {
+		if ( $header_layout !== 'normal' && in_array( $context, [ 'single-page', 'woo-page' ] ) ) {
```

Comment:
@mskapusuz, not sure it's required in this case but we usually user the strict argument set as "true" for in_array function.

Reply:
@cristian-ungureanu you're right, I've missed, thank you 🚀